README.md

```diff
@@ -325,6 +325,21 @@ If we already have a WMS GetMap URL, we can use it directly:
 
 Gain and gamma effects are also supported in these two functions as a part of `wmsParams` or `fullUrlWithWmsQueryString`.
 
+`legacyGetMapFromParams` and `legacyGetMapFromUrl` also accept the parameters that are not supported in WMS GetMap URL but are supported.
```

Comment:
Nitpick, but but this would be a lot clearer of it said smth like `legacyGetMapFromParams` and `legacyGetMapFromUrl` also accept the parameters that are not supported in WMS GetMap standard, but are supported by Sentinel Hub/sentinelhub-js.` or smth.

Reply:
Haha forgot to actually finish the sentence 😅 Thanks for spotting. And the suggestion also makes sense, will change.

Reply:
Thanks again. I paid more attention to that section of readme and changed the text a bit more.